extensions/skins/conversation_v1/player.html

```diff
@@ -79,8 +108,8 @@
             </div>
           </div>
 
-          <!-- Always show the most recent response pair, if there is one. -->
-          <div class="conversation-skin-tutor-card-bottom-section" ng-if="activeCard.answerFeedbackPairs.length > 0">
+          <!-- If the interaction is inline, always show the most recent response pair, if there is one. -->
+          <div class="conversation-skin-tutor-card-bottom-section" ng-if="activeCard.interactionIsInline && activeCard.answerFeedbackPairs.length > 0">
```

Comment:
At the moment, there is a bug with this area.

Say you're on a card with a map interaction. The grey area says (Click on the map) ->

If you click on the map and get the answer wrong, this grey area instantly disappears for a second, then instantly re-appears with the same message. This doesn't look right.

Furthermore, if you click on the map and get the answer right, this reappears as an empty thin grey area, which also doesn't look right.